packages/subgraph/schema.graphql

```diff
@@ -27,6 +27,7 @@ type ConvictionConfig @entity {
   totalStaked: BigInt!
   maxStakedProposals: Int!
   minThresholdStakePercentage: BigInt!
+  vault: Bytes
```

Comment:
shouldn't the vault config go in the higher level config of the org? maybe in a vaults level, also saying this remembering luke saying that is possible that the gardens evolves to something where you have more than one vault (Agave)